src/__snapshots__/index.test.js.snap

```diff
@@ -25,20 +25,26 @@ exports[`Ellipsis create a tooltip  and elipsis on anchor elemennt 1`] = `
     }
   >
     <div
-      className="ellipsis"
```

Comment:
I tend to  prefer working with css styles over inline styles.
working with css class is more convenient  to override